aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -51,15 +51,15 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
 
 **In-process hosting model**
 
-`CreateDefaultBuilder` calls the `UseIIS` method to boot the [CoreCLR](/dotnet/standard/glossary#coreclr) and host the app inside of the IIS worker process (*w3wp.exe* or *iisexpress.exe*). Performance tests indicate that hosting a .NET Core app in-process delivers significantly higher request throughput compared to hosting the app out-of-process and proxying requests to [Kestrel](xref:fundamentals/servers/kestrel) server.
+`CreateDefaultBuilder` adds IIS Integration Middleware by calling the `UseIIS` method to boot the [CoreCLR](/dotnet/standard/glossary#coreclr) and host the app inside of the IIS worker process (*w3wp.exe* or *iisexpress.exe*). Performance tests indicate that hosting a .NET Core app in-process delivers significantly higher request throughput compared to hosting the app out-of-process and proxying requests to [Kestrel](xref:fundamentals/servers/kestrel) server.
```

Comment:
UseIIS does not add any middleware, none are needed for the in-proc scenario. Only UseIISIntegration adds middleware for out-of-proc.

For the in-proc scenario it adds an IServer service instead.
https://github.com/aspnet/AspNetCore/blob/49c01eefecf1dbd75e5536aa803d689390c6770a/src/Servers/IIS/IIS/src/WebHostBuilderIISExtensions.cs#L40